src/resources/lang/en/crud.php

```diff
@@ -109,8 +109,13 @@
     'external_link' => 'External link',
     'choose_file' => 'Choose file',
 
+    //Unique Fields
+    'unique_error' => 'Sorry something went wrong, please check your configuration and try again',
```

Comment:
Is that a `something went wrong` error? Could this be more descriptive and more to the point if this is shown could there be a configuration error at all?